source/documentation/monitoring-an-app/application-metrics.md

```diff
@@ -1,107 +1,121 @@
 
-```yaml
-apiVersion: monitoring.coreos.com/v1
-kind: ServiceMonitor
-metadata:
-  name: my-app
-spec:
-  selector:
-    matchLabels:
-      app: my-app
-  endpoints:
-  - port: metrics
-    interval: 15s
-```
+Let's find out our current port name and number by running:
 
-More detailed information about service-monitors can be found [here](https://github.com/coreos/prometheus-operator/blob/master/Documentation/user-guides/running-exporters.md)
+```
+kubectl get svc -n <namespace>
+```
 
-#### NetworkPolicy for Monitoring Namespace
+Create and apply your service monitor `<application>-serviceMonitor.yaml`, as below:
 
+```yaml
+   apiVersion: monitoring.coreos.com/v1
+   kind: ServiceMonitor
+   metadata:
+     name: rails-app-service
+   spec:
+     selector:
+       matchLabels:
+         app: rails-app-service
+     endpoints:
+     - port: http # this is the port name you grabbed from your running service
+       interval: 15s
+   ```
+
+This will tell Prometheus to go and scrape that endpoint every 15 seconds and store any exposed metrics.
+
+#### Add a NetworkPolicy resource
+ 
 By default, all connections from outside a namespace are blocked, therefore a network policy is required for the `monitoring` namespace to be able to connect into an application namespace to scrape the metrics endpoint.
 
-Example:
+Create and apply a new resource `<application>-networkPolicy.yaml`, as below:
 
 ```yaml
-kind: NetworkPolicy
-apiVersion: networking.k8s.io/v1
-metadata:
-  name: allow-prometheus-scraping
-  namespace: my-app-namespace
-spec:
-  podSelector:
-    matchLabels:
-      app: my-app
-  policyTypes:
-  - Ingress
-  ingress:
-  - from:
-    - namespaceSelector:
-        matchLabels:
-          component: monitoring
-```
-You can view your current NetworkPolices with the following command:
+   kind: NetworkPolicy
+   apiVersion: networking.k8s.io/v1
+   metadata:
+     name: allow-prometheus-scraping
+     namespace: my-app-namespace
+   spec:
+     podSelector:
+       matchLabels:
+         app: rails-app
+     policyTypes:                                                                                                                                                         
+     - Ingress
+     ingress:
+     - from:
+       - namespaceSelector:
+           matchLabels:
+             component: monitoring
+   ```
+
+#### Querying metrics
+
+Now we've exposed our metrics and asked Prometheus to scrape application latency, we can head over to the UI and see our service appear in Prometheus [targets](https://prometheus.cloud-platform.service.justice.gov.uk/targets).
+
+Head to [Cloud Platform Prometheus](https://prometheus.cloud-platform.service.justice.gov.uk/graph) and use the following promql query to view the application latency (remembering to change the namespace value):
 
-```sh
-kubectl -n <namespace> get networkpolicies
+```
+http_server_request_duration_seconds_sum{namespace="my-namespace"}
 ```
```

Comment:
I'd suggest adding a partial screenshot, showing where in the prometheus UI the user enters the query, and what the results should look like.

Reply:
Added image